getting-started/setup-multi-node-basic.md

```diff
@@ -99,6 +62,9 @@ docs:
 [install]: /getting-started/installation
 [setup]: /getting-started/setup
 [advanced setup]: /getting-started/setup-multi-node-basic/setup-multi-node-auth
+[trust_role_setup]: /getting-started/setup-multi-node-basic/setup-multi-node-auth#multi-node-auth-trust-roles
+[password_role_setup]: /getting-started/setup-multi-node-basic/setup-multi-node-auth#multi-node-auth-password-roles
+[certificate_role_setup]: /getting-started/setup-multi-node-basic/setup-multi-node-auth#multi-node-auth-certificate-roles
```

Comment:
Minor point, but should we rename these:

`getting-started/setup-multi-node` and `getting-started/setup-multi-node/auth` ?

Reply:
It looks like there's already a zombie setup-multi-node page still lingering in this directory that was split into these two new pages (setup-multi-node-basic and setup-multi-node-auth).  @solugebefola did this refactoring about three weeks back, so I'd like to get his feedback here before mucking things up too much.